src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx

```diff
@@ -0,0 +1,150 @@
+    const {
+      yearAbandoned,
+      firstTreePlanted,
+      plantingDensity,
+      maxPlantingDensity,
+      plantingSeasons,
+    } = metadata;
+
+    return Boolean(
+      yearAbandoned ||
+        firstTreePlanted ||
+        plantingDensity ||
+        maxPlantingDensity ||
+        (plantingSeasons && plantingSeasons.length > 0)
+    );
+  }, [metadata]);
+
+  const shouldRenderAdditionalInfo = useMemo(() => {
+    const {
+      mainChallenge,
+      siteOwnerName,
+      longTermPlan,
+      acquisitionYear,
+      motivation,
+    } = metadata;
+    return Boolean(
+      mainChallenge ||
+        siteOwnerName ||
+        (isTreeProject &&
+          metadata.siteOwnerType &&
+          metadata.siteOwnerType?.length > 0) ||
+        (isTreeProject && metadata.degradationCause) ||
+        longTermPlan ||
+        acquisitionYear ||
+        motivation
+    );
+  }, [metadata]);
+
+  const location = useMemo(() => {
+    if (!tpo.address) return '';
+    const { address, zipCode, city, country } = tpo.address;
+    return [
+      address,
+      zipCode,
+      city,
+      country
+        ? tCountry(tpo.address.country.toLowerCase() as Lowercase<CountryCode>)
+        : '',
+    ]
+      .filter(Boolean)
+      .join(',');
+  }, [tpo]);
+
+  const shouldRenderProjectDownloads = useMemo(() => {
+    return certificates.length > 0 || expenses.length > 0;
+  }, [certificates, expenses]);
+
+  return (
+    <section className={styles.projectInfoContainer}>
+      {reviews?.length > 0 && <ProjectReview reviews={reviews} />}
+      {description && <AboutProject description={description} wordCount={60} />}
+      {videoUrl && <VideoPlayer videoUrl={videoUrl} />}
+      {images.length > 0 && (
+        <ImageSlider images={images} type={'project'} isMobile={isMobile} />
+      )}
+      {isMobile && <MapPreview setSelectedMode={setSelectedMode} />}
+      {shouldRenderKeyInfo && (
+        <KeyInfo
+          abandonment={isTreeProject ? metadata.yearAbandoned : null}
+          firstTree={isTreeProject ? metadata.firstTreePlanted : null}
+          plantingDensity={isTreeProject ? metadata.plantingDensity : null}
+          maxPlantingDensity={
+            isTreeProject ? metadata.maxPlantingDensity : null
+          }
+          employees={metadata.employeesCount}
+          plantingSeasons={isTreeProject ? metadata.plantingSeasons : null}
+        />
+      )}
+      {shouldRenderAdditionalInfo && (
+        <AdditionalInfo
+          mainChallengeText={metadata.mainChallenge}
+          siteOwnershipText={metadata.siteOwnerName}
+          siteOwnershipType={isTreeProject ? metadata.siteOwnerType : null}
+          causeOfDegradationText={
+            isTreeProject ? metadata.degradationCause : null
+          }
+          whyThisSiteText={metadata.motivation}
+          longTermProtectionText={metadata.longTermPlan}
+          acquiredSince={metadata.acquisitionYear}
+        />
+      )}
```

Comment:
1. Need to clarify all information that can be seen in KeyInfo and AdditionalInfo sections, and update accordingly.
2. Conditions to render these sections will also need updating in that case.